rust/tools/authorization-logic/src/parsing/AuthLogic.g4

```diff
@@ -25,20 +25,25 @@ grammar AuthLogic;
 //-----------------------------------------------------------------------------
 // Parser
 //-----------------------------------------------------------------------------
+id 
```

Comment:
BTW, I thought we just had one copy of `Authlogic.g4`? Are we starting to diverge between C++ and rust implementations?

Reply:
Do we? I think I missed how exactly that was set up. Is there some script that copies from this file to some other location, or does it work some other way?

Reply:
@harsha-mandadi can comment on above, but I am guessing it was manually copied. Can we shift to use a single central version as part of this PR (or a smaller separate single one)?

Reply:
`AuthLogic.g4` was copied manually. I made changes to rust build to point to a C++ version of g4 file PR #469

Reply:
It looks like the c3po Copybara commit in GH didn't include the rust-side file edit for some reason, from briefly looking at: https://github.com/google-research/raksha/commit/b2729ec8806d16c222e402eef34bb33981a5d726

The PR description is potentially ambiguous/confusing in that it states "move", but if I am reading it correctly, it did not actually delete the original `AuthLogic.g4` file which would have been preferable unless there was a known reason to leave it?

Reply:
AuthLogic.g4 rust version should have been deleted.

Reply:
Just a guess, the Raksha `copy.bara.sky` file looks like it's skipping auth logic via this part of config (`exclude` in the github files section, line 33):

```
        # Don't need auth logic yet.
        "rust/**",
```

Maybe this led to stripping the changes to auth logic as we used `ready-to-pull` label, so it imported, then exported.

Do we merge all changes through that flow, or can/do we also land them in GH directly?